site2/website/blog/2021-05-13-Apache-Pulsar-2-7-2.md

```diff
@@ -0,0 +1,68 @@
+---
+author: Enrico Olivelli
+authorURL: https://twitter.com/eolivelli
+title: Apache Pulsar 2.7.2
+---
+We are very glad to see the Apache Pulsar community has successfully released 2.7.2 version.
+This is a minor release that introduces stability fixes and a few new features without breaking changes.
+
+<!--truncate-->
+
+### News and noteworthy
+
+Here is a selection of the most interesting and major features added to Pulsar 2.7.2.
+
+- Improvements in stability in the Kinesis Connector [#10420](https://github.com/apache/pulsar/pull/10420)
+- Improvement in passing ENV variables to the Bookie (PULSAR_EXTRA_OPTS) [#10397](https://github.com/apache/pulsar/pull/10397)
+- Allow the C++ client to be built in Windows and add CI for verification [#10387](https://github.com/apache/pulsar/pull/10387)
+- Allow to activate every BookKeeper Client features in the Broker [#9232](https://github.com/apache/pulsar/pull/9232)
+- Improvements Pulsar Proxy
+- Upgrade core networking libraries: Jetty and Netty 
+
+[Here](https://github.com/apache/pulsar/pulls?page=1&q=is%3Apr+label%3Arelease%2F2.7.2]) you can find the list of all the improvements and bugfixes.
+
+### Contributors for 2.7.2 release
+
+We would like to thank all the contributors for this release.
+As every healthy Open Source Project, Apache Pulsar is great because it is supported by a vibrant community.
+
+Code contributors (names taken from GitHub API):
+- Ali Ahmed
+- Andrey Yegorov
+- Binbin Guo
+- David Kjerrumgaard
+- Deon van der Vyver
+- Devin Bost
+- Enrico Olivelli
+- Guangning E
+- Kevin Wilson
+- Lari Hotari
+- Marvin Cai
+- Masahiro Sakamoto
+- Matteo Merli
+- Michael Marshall
+- Rajan Dhabalia
+- Shen Liu
+- Ting Yuan
+- Vincent Royer
+- Yong Zhang
+- Yunze Xu
+- Zhanpeng Wu
+- Zike Yang
+- baomingyu
+- CongBo
+- dockerzhang
+- feynmanlin
+- hangc0276
+- li jinquan
+- limingnihao
+- linlinnn
+- mlyahmed
+- PengHui Li
+- ran
+
+Also, we want to thank everyone who spent his time to report issues and to share his story about using Pulsar.
```

Comment:
```suggestion
Also, we want to thank everyone who spent his time reporting issues and sharing the story about using Pulsar.
```

Use gender-neutral language to avoid gender bias.

Reply:
interesting. good to know. I will keep in mind for the next time.